k8s/cmd/admission-webhook/main.go

```diff
@@ -136,20 +136,44 @@ func (whsvr *WebhookServer) mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 	logrus.Infof("AdmissionReview for Kind=%v, Namespace=%v Name=%v UID=%v patchOperation=%v UserInfo=%v",
 		req.Kind, req.Namespace, req.Name, req.UID, req.Operation, req.UserInfo)
 
-	var deployment appsv1.Deployment
-	if err := json.Unmarshal(req.Object.Raw, &deployment); err != nil {
-		logrus.Errorf("Could not unmarshal raw object: %v", err)
+	var meta *metav1.ObjectMeta
```

Comment:
I would like to see this file changed in a separate PR with a better explanation of what we do here.
This PR is about adding to the CI and then also chnages the core component.
We were already burned by merging huge patches with too many changes incorporated. Let's try to be more strict.